Tests/Factory/DiactorosFactoryTest.php

```diff
@@ -75,6 +75,7 @@ public function testCreateRequest()
         $psrRequest = $this->factory->createRequest($request);
 
         $this->assertEquals('Content', $psrRequest->getBody()->__toString());
+        $this->assertEquals('Content', $request->getContent()); // check if getContent() can be called multiple times
```

Comment:
I would have moved this into a dedicated test (named something like `testGetContentCanBeCalledAfterRequestCreation`).